### PR TITLE
bbuffs dmr and br

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
br gets 40 round magazines
dmr gets slightly faster wield delay and less aim mode slowdown

## Why It's Good For The Game
battle rifle is bad, it should've had double the mag capacity from the start
dmr is a bit bad, it could use some minor love

## Changelog
:cl: 
balance: dmr has slightly less slowdown, less aim slowdown, and battle rifle has had its capacity upped from 25 to 35
/:cl:

### CODE REVIEW

**Comment by Tupinambis** [comment]
The BR not having a reasonable magazine size was my main issue with it. If it could have a 3 round burst option that would be even better. Could even have 36 rounds to a mag.

File: code/modules/projectiles/magazines/rifles.dm [hunk 1]
```diff
@@ -104,7 +104,7 @@
 	icon_state = "t64"
 	w_class = WEIGHT_CLASS_NORMAL
 	default_ammo = /datum/ammo/bullet/rifle/standard_dmr
-	max_rounds = 25
```
Comment (TiviPlus) [inline code/modules/projectiles/magazines/rifles.dm:107]: 35

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Fix your CL

**Comment by lKiseki** [comment]
Good stuff. These weapons needed some love. Should be given the test merge candidate flag so we can see it in action.

File: code/modules/projectiles/magazines/rifles.dm [hunk 2]
```diff
@@ -104,7 +104,7 @@
...
+	max_rounds = 40
```
Comment (TiviPlus) [inline code/modules/projectiles/magazines/rifles.dm:107]:
```suggestion
	max_rounds = 35
```

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**